# HYPOTHESIS TESTING

Hypothesis testing is a fundamental statistical method used to make inferences about population parameters based on sample data. It involves formulating two competing hypotheses: the null hypothesis (H0), which typically states that there is no effect or no difference between groups, and the alternative hypothesis (H1), which asserts the presence of an effect or difference. The process generally entails selecting an appropriate statistical test based on the nature of the data and the research question, calculating a test statistic from the sample data, and determining the probability of obtaining such results under the assumption that the null hypothesis is true, known as the p-value. If the p-value is below a predetermined significance level (usually 0.05), the null hypothesis is rejected in favor of the alternative hypothesis, suggesting evidence for the presence of an effect or difference. Otherwise, if the p-value is above the significance level, the null hypothesis is not rejected. Hypothesis testing provides a systematic framework for drawing conclusions from data, aiding researchers in making informed decisions and drawing valid inferences about the population.

### TASKS

#### Q1.Suppose a child psychologist claims that the average time working mothers spend talking to their children is at least 11 minutes per day. You conduct a random sample of 1000 working mothers and find they spend an average of 11.5 minutes per day talking with their children. Assume prior research suggests the population standard deviation is 2.3 minutes.Conduct a test with a level of significance of alpha = 0.05.

#Null Hypothesis

HO=µ≤11 minutes



#Alternative Hypothesis

Ha=µ>11 minutes   

x̄=11.5 minutes

n=1000

σ=2.3 minutes

α = 0.05

#### Statistical Test:
Since we have a sample of 1000, which is greater than 30, and we have a sample mean, a known population standard deviation, and want to compare the sample mean to a population value, we'll use a z-test.

rejection criteria is z_statistic>critical value , fail to accept Ho

Formula Z=(x̄-µ)/(σ/√n)

Where:
x̄ = sample mean (11.5 minutes), 
μ = population mean (11 minutes), 
σ = population standard deviation (2.3 minutes), 
n = sample size (1000)

In [4]:
import math

In [7]:
Z=(11.5-11)/(2.3/math.sqrt(1000))
Z

6.874516652539955

#### Determine Critical Value or P-value:

Since we're conducting a one-tailed test (since we're testing if the average time is more than 11 minutes), we'll look up the critical value for a z-score corresponding to an alpha level of 0.05 from z-distribution table

#corresponds to an alpha level of 0.05.

Critical value=1.645 

#### Conslusion:
Since our calculated z-score (6.88) is much greater than the critical value (1.645), we reject the null hypothesis.
We have sufficient evidence to conclude that the average time working mothers spend talking to their children is indeed more than 11 minutes per day
.

#### Q2. A coffee shop claims that their average wait time for customers is less than 5 minutes. To test this claim, a sample of 40 customers is taken, and their wait times are recorded. The sample mean wait time is found to be 4.6 minutes with a standard deviation of 0.8 minutes. Perform a hypothesis test at a significance level of 0.05 and determine whether there is enough evidence to support the coffee shop's claim.

#Null Hypothesis

H0=µ≥5 minutes

#Alternative Hypothesis

Ha=µ<5 minutes


Sample size (n) = 40

Sample mean (x̄) = 4.6 minutes

Sample Standard deviation (σ) = 0.8 minutes

Population mean (μ) = 5 minutes

Significance level (α) = 0.05 (5%)

degrees of freedom=n-1=39


#### Statistical Test:
In cases where we're unsure about the population standard deviation, opting for a t-test is appropriate. . When dealing with sample sizes that exceed 30, the t-test remains applicable even in cases where the population standard deviation is unknown. This is facilitated by the Central Limit Theorem, which asserts that as the sample size increases, the distribution of sample means tends towards a normal distribution.

Formula t=(x̄-µ)/(σ/√n)

Where: x̄ = sample mean (4.6 minutes), μ = population mean (5 minutes), σ = population standard deviation (0.8 minutes), n = sample size (40)

In [1]:
import math
t=(4.6-5)/(0.8/math.sqrt(40))
t

-3.162277660168382

Now, we'll compare this t-value to the critical t-value at a significance level of 0.05 with degrees of freedom (df)

### Determine Critical Value or P-value:
For a one-tailed test at α = 0.05 and df = 39, the critical t-value is approximately -1.685 (since we're testing for less than).

#corresponds to an alpha level of 0.05.

Critical value=-1.645

### Conslusion:
Since our calculated z-score (-3.16) is much smaller than the critical value (-1.645), we reject the null hypothesis.
We have sufficient evidence to conclude that the average wait time for customers at the coffee shop is indeed less than 5 minutes.